dev-environment.yml

```
@@ -0,0 +1,9 @@
+name: hist
+channels:
+  - conda-forge
+dependencies:
+  - pip >=18
+  - root
+  - pip:
+    - -r dev-requirements.txt
+    - -e .[dev]
```

```suggestion
    - -e .
```

If there are any dev requirements, these need to go in as conda requirements if at all possible, not as pip ones.

---

Note that this is really here just to support the tests with ROOT.